src/game_screens/game_screen.py

```diff
@@ -9,23 +9,25 @@
 from src.objects.coin import Coin
 from src.objects.bluecoin import BlueCoin
 from src.objects.bomb import Bomb
+       
 
 
+##game_manager=Game_manager()
 class Game_screen(Screen):
-    def __init__(self, pygame, res, surface, size, gameclock, game_manager):
+    def __init__(self, pygame, res, surface, size, gameclock, game_manager):    
         Screen.__init__(self, pygame, res, surface)
-
+           
         # set up initial variables
         self.need_reset = False
         self.size = size
-        self.result = 0
-        self.i = 1
+	self.result = 0
```

Comment:
indentation problems like this

Reply:
@amrit-choudhary  Everything is indented properly in my code but it shows these errors on github ..
Can you help me?

Reply:
can you show a screenshot from your editor of the same code?

Reply:
![image](https://user-images.githubusercontent.com/35529399/49857995-e25a3400-fe19-11e8-97a4-85fb85ba99df.png)

Reply:
is it related to push ??

Reply:
I don't know.
I think it's related to the editor.
[link](https://stackoverflow.com/questions/29074158/indentation-inconsistency-python-taberror-different-editors)
I use VS Code.

Reply:
@kartikct25 I use Atom, but that's beside the point I'm going to make: there should be options in most of the latest editors (like Atom, which is why I gave my example) that deal with whitespaces and other formatting things etc. That should save you from manually checking it out